.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -48,6 +48,7 @@ body:
         - Docker
         - Fly.io
         - NAS
+        - Electron App
```

Comment:
```suggestion
        - Desktop App (Electron)
```

A lot of folks wont know what "electron" is. All they know is they have a program on their computer.